CONTRIBUTING.md

```diff
@@ -24,20 +24,18 @@ serving the tests of Nmstate.
 
 - `./examples/` - Contains YAML examples for different configurations.
 
-- `./libnmstate/ifaces/` Contains the classes including the attributes of the 
+- `./rust/src/lib/ifaces/` Contains the classes including the attributes of the 
 interfaces.
  
-- `./libnmstate/nispor/` Contains the nmstate nispor plugin, currently for 
+- `./rust/src/lib/nispor/` Contains the nmstate nispor plugin, currently for 
 querying kernel network status. 
 
-- `./libnmstate/nm/` Contains the nmstate NetworkManager plugin, currently for 
+- `./rust/src/lib/nm/` Contains the nmstate NetworkManager plugin, currently for 
 querying network status and applying network states. 
 
-- `./libnmstate/plugins/` Contains the plugins supported by default (e.g. ovsdb). 
+- `./rust/src/lib/ovsdb/` Contains ovsdb plugin supported by default. 
```

Comment:
Same here. We do not have plugins anymore. A better definition would be `Contains the code related to ovsdb communication and structures`

Reply:
ok

Reply:
changed